distributions/demo-google-deployment/api/build.gradle

```diff
@@ -57,35 +58,73 @@ dependencies {
     compile project(':extensions:auth:portability-auth-instagram')
 }
 
-// TODO: Include index.html in jar
 
 addCloudExtensionDependency(project)
 
 mainClassName = 'org.dataportabilityproject.api.ApiMain'
 
+task copyEnvResources(type: Copy) {
+    description = 'Copies environment-specific resources and configuration into the jar, ' +
```

Comment:
Could we eventually remove index.html from being copied in case the deployment is using rpc instead of http

Reply:
I added some comments to the README clarifying how this works and how this is just one way DTP may be deployed on GCP. I think the RPC stuff will need further discussion so I'll leave it out of this demo google deployment (perhaps that could be another demo google deployment later on).